Documentation/contributing/maintainers.md

```diff
@@ -0,0 +1,44 @@
+
+Maintainers may edit PR and issue titles to improve readability and/or accuracy. GitHub provides history for title changes.
+
+Maintainers may edit your comments if and only if the edit adds substantially to the readability or usability of the comment and doesn't change the content. The maintainer will add a short descriptive note with their @handle to explain their change at the end of your comment (for example: "_Edited by @richlander -- added call-stack formatting_"). 
+
+Maintainers may make the following changes to contributor comments:
+
+- Improving markdown formatting - Changes limited to [single/triple-ticks](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code) for code and call-stacks, broken markdown links, and broken formatting (but not adding formatting or altering spelling or grammar).
+- Adding "fixes link" - GitHub automatically closes issues from a PR if a [fixes link](https://help.github.com/articles/closing-issues-via-commit-messages/) is part of a commit message or initial PR comment. The addition of the link makes it easier to understand the relationship between issues and PRs.
+- Adding links to latest or related content - The addition of a link to another issue or PR or another comment significantly improves readability (for example: "_Edited by @richlander -- latest version of proposal at URL_"). The link can be placed at the top and/or bottom the existing comment.
+
+## Reporting Code of Conduct Violations
+
+All .NET repositories have adopted a code of conduct, either the [.NET Foundation Code of Conduct](https://dotnetfoundation.org/code-of-conduct) or the [Microsoft Code of Conduct](https://opensource.microsoft.com/codeofconduct/). Most have adopted the former code of conduct since most .NET repos are part of the .NET Foundation.
+
+Maintainers will report potential code of conduct violations they observe to the appropriate code of conduct review group at either conduct@dotnetfoundation.org (for [dotnet org](https://github.com/dotnet) and [aspnet org](https://github.com/aspnet)) or opencode@microsoft.com (for [microsoft org](https://github.com/microsoft)). You are encouraged to report concerns via these same email addresses. Don't worry which one you use (or both); the concern will end up in the right place.
+
+The Code of Conduct team will delete comments that are considered code of conduct violations. Maintainers will unilaterally delete comments that are objectively severe violations of the code of conduct and submit the original comment to the Code of Conduct team after-the-fact for review.
+
+## Summary of Maintainer Code
```

Comment:
I wonder if this would make more sense at the start of the document, kind of like a TL;DR ?